components/fxa-client/src/internal/send_tab.rs

```diff
@@ -27,7 +27,7 @@ impl FirefoxAccount {
     }
 
     fn load_or_generate_keys(&mut self) -> Result<PrivateSendTabKeys> {
-        if let Some(s) = self.state.commands_data.get(send_tab::COMMAND_NAME) {
+        if let Some(s) = self.state.send_tab_key() {
```

Comment:
What do you think about keeping the `state` oblivious of "send tab" and have it return and store "command_data"

Reply:
Maybe a helper in this module that is called `get_key` that calls into the `command_data`

Reply:
Makes sense to me, I'll update this.